### PR TITLE
Use Node Env for Jest

### PR DESCRIPTION
**Pull request by michael-markl**
This is the alternative to #777 . @maxammann Push the green button, if you are more convinced of this :D I just need one working approach :D

### CODE REVIEW

File: administration/package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
   "scripts": {
     "start": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts start",
     "build": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts build",
-    "test": "react-scripts test",
+    "test": "node --experimental-global-webcrypto node_modules/jest/bin/jest.js",
```
Comment (michael-markl) [inline administration/package.json:55]: Necessary until Node 19. Once we upgrade node to the next LTS version 22 (which has not released yet), can simplify this to just 
```suggestion
    "test": "jest",
```

**Comment by maxammann** [comment]
For me both solutions are find :) 

The original one is more flexible but more complex. This one is simpler but more constraint.

Sorry, bur deciding that isone of the hardest choices :D

**Comment by michael-markl** [comment]
@maxammann Feel free to approve up one or both of them, and I will decide then :P (Currently still tending towards the karma solution)

**Review by maxammann — APPROVED** [review]

**Comment by michael-markl** [comment]
Closing in favor of #777

**Pull request closed without merging**